.env

```diff
@@ -0,0 +1,2 @@
+RECORD_TYPE = "Wikipedia"
```

Comment:
Typically you'll want multiple .env files. One with non-secret information you check in and then another with secrets (`.env.local` or `.env.secret` or similar). `dotenv` will load them all in some overwrite order.